docs/source/manim_directive.py

```diff
@@ -265,6 +278,57 @@ def run(self):
         return []
 
 
+rendering_times_file_path = "../rendering_times.csv"
+
+
+def _write_rendering_stats(scene_name, run_time, file_name):
+    line = ",".join(
```

Comment:
it's a csv file right? Please use CSV https://docs.python.org/3/library/csv.html module :)